OpenRA.Mods.TS/Traits/Buildings/LowBridge.cs

```diff
@@ -20,7 +20,7 @@ namespace OpenRA.Mods.TS.Traits
 	[Desc("Bridge actor that can't be passed underneath.")]
 	class LowBridgeInfo : ITraitInfo, Requires<BuildingInfo>
 	{
-		public readonly string TerrainType = "Road";
+		public readonly string TerrainType = "Bridge";
```

Comment:
You need to define actor speeds for this new terrain type, currently this makes low bridges unpassable by all actors.